models/coinmarketcap.py

```diff
@@ -26,15 +26,25 @@ def __repr__(self):
 class CoinmarketcapAPI(Fetcher):
     _COINMARKETCAP_URL = 'https://api.coinmarketcap.com/v1/ticker?limit={}'
 
+    # Coins that doesnt have
```

Comment:
@bashalex this used to be a list for coins that does not have proper info (e.g. `{"update_time": none}`). I've moved from this, but it is somehow got here.
Fixed in 9871d9d